setup.cfg

```diff
@@ -1,5 +1,21 @@
 [flake8]
-exclude = build,.git,build-pypy,.tox
+exclude =
+    .git,
+    __pycache__,
+    docs/source/conf.py,
```

Comment:
There is no such file in dulwich

Reply:
Yes, this directory is not available, it is not necessary.

Reply:
I think it is possible that some of the contents of this project are not structured according to the directory structure of most projects for technical manuals, but I am not sure about this, but most of the `conf.py` I found will be in `docs/ Under the source/` directory.

So I want to ask why is it different?